python/ray/serve/api.py

```diff
@@ -301,7 +302,9 @@ class MyDeployment:
         user_config: Config to pass to the reconfigure method of the deployment. This
             can be updated dynamically without restarting the replicas of the
             deployment. The user_config must be fully JSON-serializable.
-        max_concurrent_queries: Maximum number of queries that are sent to a
+        max_concurrent_queries: [DEPRECATED] Maximum number of queries that are sent to
+            a replica of this deployment without receiving a response. Defaults to 100.
+        max_ongoing_requests: Maximum number of requests that are sent to a
             replica of this deployment without receiving a response. Defaults to 100.
```

Comment:
```suggestion
        max_ongoing_requests: Maximum number of requests that are sent in parallel
            to each replica of this deployment. The limit is enforced across all callers
            (HTTP requests or DeploymentHandles). Defaults to 100.
```
Might as well fly-by improve it! Let me know if you have better wording suggestions.